students/HABTAMU/session07/test_html_output7.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
you generally don't want to put output like this into git.

always check carefully what is being added when you commit -- do a "git status" first!

Reply:
you want that DOCTYPE tag, yes,

but you don't want to put generated files, like `test_html_output7.html` in git.